dockstore-common/src/main/java/io/dockstore/common/DescriptorLanguage.java

```diff
@@ -50,7 +50,12 @@ public boolean isRelevantFileType(FileType type) {
             return super.isRelevantFileType(type) || type == FileType.DOCKERFILE || type == FileType.NEXTFLOW;
         }
     },
-    SERVICE("service", "generic placeholder for services", FileType.DOCKSTORE_SERVICE_YML, FileType.DOCKSTORE_SERVICE_TEST_JSON, true);
+    SERVICE("service", "generic placeholder for services", FileType.DOCKSTORE_SERVICE_YML, FileType.DOCKSTORE_SERVICE_TEST_JSON, true) {
+        @Override
+        public boolean isRelevantFileType(FileType type) {
+            return super.isRelevantFileType(type) || type == FileType.OTHER;
```

Comment:
Is FileType.OTHER used by anything other than services? 
Maybe it should be FileType.DOCKSTORE_SERVICE_OTHER or FileTYPE.DOCKSTORE_SERVICE_MISC if not.

Reply:
Currently only services, though I could see it as a default value used by other entities.

Reply:
So I guess that's a question. What are the situations we would want assorted files to be intermingled with other types (tools, workflows) and when would we not want them to be?

Reply:
Going with DOCKSTORE_SERVICE_OTHER